theme/imagecaption/theme.scss

```diff
@@ -2,27 +2,14 @@
 // For licensing, see LICENSE.md or http://ckeditor.com/license
 
 @import '~@ckeditor/ckeditor5-theme-lark/theme/helpers/_colors.scss';
-@import '~@ckeditor/ckeditor5-theme-lark/theme/helpers/_shadow.scss';
-@import '~@ckeditor/ckeditor5-theme-lark/theme/helpers/_states.scss';
+@import '~@ckeditor/ckeditor5-theme-lark/theme/helpers/_spacing.scss';
+@import '~@ckeditor/ckeditor5-theme-lark/theme/helpers/_fonts.scss';
 
 .ck-widget.image {
```

Comment:
Isn't `.ck-editor .image` enough? Because I think it is and it's a better DX because devs don't need to think what is "editable" and why. All they want is a styled **image** in the **CKEditor**.